docs/passphrases.rst

```diff
@@ -85,3 +84,13 @@ journalist will require the following passphrases:
 The backup that is created during the installation of SecureDrop is also
 encrypted with the application's GPG key. The backup is stored on the
 persistent volume of the Admin Live USB.
+
+*Transfer Device* and *Export Device*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As noted in the :doc:`setup guide <set_up_transfer_and_export_device>`,
+we recommend using encrypted USB drives for transferring files to and from the
+*Secure Viewing Station*. Journalists performing these operations will need
+access to the encryption passphrase. Because the same passphrase has to be used
+on the source and the destination, we recommend storing this passphrase in the
+journalist's own existing password manager (e.g., on their mobile phone).
```

Comment:
Nit: "source and destination" is somewhat confusing here, given the use of the term "source" to denote user roles in the SecureDrop workflow. Perhaps rephrase as:

> Because the same passphrase has to be used frequently on multiple workstations (i.e., *Journalist Workstation* and *Secure Viewing Station*)

As far as the recommendation for using an individual password manager, rather than Tails's own KeePassX setup, that mostly makes sense for the _Export_ device, not the _Transfer_ device—but fine with the change to keep things consistent. The "Passphrase Best Practices" guide makes no mention of using an individual password manager, however—might be worth updating that guidance in line with these changes.

Reply:
Good point re: confusing language, tried to clarify that. 

Re: the Transfer Device, one other concern about stashing that in KeePassX is making sure that it's always updated on two devices, without any automatic mechanism to keep them in sync. If those two password managers are out of sync, the journalist may complete a copy operation halfway, only to find that they don't have access to the cred when they need it. 

If they pull it from their smartphone's password manager, it'll either work on the *Journalist Workstation* or it won't work at all, reducing the likelihood of wasted time. So I think more than just a matter of consistency, having only a single copy of the passphrase is generally the better recommendation.

For orgs with many journalists, having it in KeePassX twice may be a better option (even if that won't work for the *Export Device*), but our docs also suggest that in those cases, it may be good to have multiple Transfer Devices. This idea that the Transfer Device/Export Device is _personal_ to the journalist is I think worth exploring further (especially as an alternative to the impractical and half-hearted suggestion to use GPG), but perhaps out of scope for this PR.